corehq/apps/app_manager/static/app_manager/js/app_diff.js

```diff
@@ -82,10 +80,11 @@ hqDefine('app_manager/js/app_diff.js', function () {
          *
          * When called renders a diff into the $el element
          */
-        self.renderDiff = function() {
+        self.renderDiff = function(appIdOne, appIdTwo) {
```

Comment:
this enables use to reused the cached responses to the server because we don't have to instantiate a new AppDiff object for every diff we create